distribution/src/main/docker/nginx/lowcodeluna.conf

```diff
@@ -0,0 +1,39 @@
+# Copyright (C) 2022 Dynamic Solutions
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+server {
+    listen       80;
+    listen       84 default_server;
+    server_name  low-code.luna.127.0.0.1.nip.io;
+    index        Homepage.html;
+
+    location /health {
+        access_log off;
+        add_header  Content-Type    text/html;
+        return 200;
+    }
+
+    location /apps/bulma/web_root {
```

Comment:
We need a similar configuration for `/apps/luna/web_root` because custom styles won't work on the published page without it.